build.gradle

```diff
@@ -165,19 +174,26 @@ subprojects {
   }
 
   // Findbugs
-  apply plugin: 'findbugs'
-  findbugs {
-    toolVersion = "3.0.1"
+  apply plugin: 'com.github.spotbugs'
+  spotbugs {
+    toolVersion = "3.1.3"
     reportLevel = "low"
     effort = "max"
+    includeFilter = file("findbugs-exclude.xml")
```

Comment:
Minor nitpick but this file should probably be called `spotbugs-exclude.xml`

Reply:
SpotBugs & FindBugs are still interestingly joined, note that the root element of the content of the exclude xml is <FindBugsFilter> I think the ideas is to make it as easy as possible to migrate over.